components/brave_vpn/resources/panel/container.tsx

```diff
@@ -80,9 +80,10 @@ function Main () {
       onConnectionStateChanged: (state: ConnectionState) => {
         if (state === ConnectionState.CONNECT_FAILED) {
           setHasError(true)
-          setOn(false)
         }
 
+        setOn(state == ConnectionState.CONNECTED ||
+              state === ConnectionState.CONNECTING);
```

Comment:
Why does the first one has two equal operator vs the second one with three?

Reply:
it's mistake :)